reana_cluster/backends/kubernetes/templates/deployments/workflow-controller-template.yaml

```diff
@@ -37,6 +36,8 @@ spec:
         - containerPort: 5000
           name: http
         env:
+        - name: K8S_REANA_SERVICE_ACCOUNT_NAME
+          value: {{REANA_SYSTEM_SERVICE_ACCOUNT}}
```

Comment:
Can we harmonise the two names?  (`REANA_SERVICE...` and `REANA_SYSTEM_SERVICE...`)